containers/dotnet/definition-manifest.json

```diff
@@ -7,6 +7,14 @@
 		"tags": [
 			"dotnet:${VERSION}-${VARIANT}"
 		],
+		"architectures": {
+			"6.0-focal": ["linux/amd64", "linux/arm64"],
```

Comment:
btw, from learnings on other containers, iirc focal has issues with arm64 (qemu), and so linux/arm64 was being intentionally skipped on [focal](https://github.com/microsoft/vscode-dev-containers/issues/558#issuecomment-907523792)

Reply:
@dnastrain , good point. So far I understand this is not aiming `qemu`, but it would be better to make sure it runs `focal` on M1.

Reply:
I had the same question.  It seems like the repo avoids building ARM64 for `focal` and `buster` pretty consistently, so worth understanding if that applies here.

I was also wondering if it makes sense to add `hirsute` to allow for an Ubuntu LTS with ARM64 emulation support and a longer support period.

All probably questions for Chuck

Reply:
Hey @bderusha , right the assumption here is that focal is supported for arm64 (.ie [6.0-focal-arm64v8](https://github.com/dotnet/dotnet-docker/blob/a5b98190e6b752e2967734b84033c1ec90ec855b/src/sdk/6.0/focal/arm64v8/Dockerfile)), but probably other containers don't have it available.
With regards `hirsute` it is not listed even for x86.

Reply:
Ah I am caught up now.  I didnt see that these use the published dotnet image as the base container and not the vanilla OS base.  Carry on! 🎸